packages/inline-svg/index.js

```diff
@@ -1,3 +1,9 @@
-export default function InlineSvg({ src, ...props }) {
-  return <div dangerouslySetInnerHTML={{ __html: src }} {...props}></div>
+const supportedElements = {
+  div: 'div',
+  span: 'span',
+}
+
+export default function InlineSvg({ wrapper, src, ...props }) {
```

Comment:
```suggestion
export default function InlineSvg({ wrapper = 'div', src, ...props }) {
```

Have you tried this w/ `yalc` yet? I think TS will complain about `wrapper` if you try not to define it in the implementation w/o a default or explicit optional in an interface

Reply:
Not familiar with `yalc`. Not seeing it in scripts, is this a tool outside of the project I should be using?